<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicMusicPlayer</class>
 <widget class="QMainWindow" name="BasicMusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BasicMusicPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>117</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="4" rowspan="2">
       <widget class="QSlider" name="sliderVolume">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="bFind">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="bPlay">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string>|&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lVolume">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="5" rowspan="2">
       <widget class="QListWidget" name="lPlaylist">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>170</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="bForward">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;|</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="bBackward">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string>|&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
